plugins/src/ios/InitializePluginManager.m

```diff
@@ -0,0 +1,28 @@
+//
+//  InitializePluginManager.m
+//  Example_App
+//
+//  Created by Barak Ben Hur on 03/08/2022.
+//
+
+#import "InitializePluginManager.h"
+#import "Example_App-Swift.h"
+
+@implementation InitializePluginManager
+
++ (void) load {
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        [NSNotificationCenter.defaultCenter addObserver: self
+                                               selector: @selector(didFinishLaunching:)
+                                                   name: UIApplicationDidFinishLaunchingNotification
+                                                 object: nil];
+    });
+}
+
++ (void) didFinishLaunching: (NSNotification*) n {
+    Optimove_Cordova *plugin = [[Optimove_Cordova alloc] init];
+    [plugin didFinishLaunching:n];
```

Comment:
Is there a way we can make this method on Optimove_Cordova static so we avoid creating two Cordova plugin instances?

Reply:
done